test/services/EmploymentServiceSpec.scala

```diff
@@ -25,22 +25,25 @@ import utils.TestUtils
 
 import scala.concurrent.Future
 
-class CreateEmploymentServiceSpec extends TestUtils {
+class EmploymentServiceSpec extends TestUtils {
 
-  val mockConnector = mock[CreateEmploymentConnector]
-  val createEmploymentService = new CreateEmploymentService(mockConnector)
+  val mockCreateEmploymentConnector = mock[CreateEmploymentConnector]
```

Comment:
These two vals for the connectors give warnings about needing type annotation, if you wanna take care of them?

Reply:
will do